desc/compute/geom_utils.py

```diff
@@ -28,6 +28,10 @@ def reflection_matrix(normal):
 def rotation_matrix(axis, angle=None):
     """Matrix to rotate points about axis by given angle.
 
+    NOTE: not correct if a and b are antiparallel, will
```

Comment:
dont think this comment applies here

Reply:
I am certain it does, I checked it and the rotation matrix is wrong for this case with this function too